src/NzbDrone.Core/Update/InstallUpdateService.cs

```diff
@@ -170,7 +182,12 @@ public void Execute(ApplicationUpdateCommand message)
 
         public void Execute(InstallUpdateCommand message)
         {
-            InstallUpdate(message.UpdatePackage);
+            var success = InstallUpdate(message.UpdatePackage);
+
+            if (!success)
+            {
+                throw new NzbDroneClientException(System.Net.HttpStatusCode.Conflict, "Failed to install update");
```

Comment:
Not sure if conflict is the right status code here, but offhand I can't think of a better one.